spec/unit/functions/epp_spec.rb

```diff
@@ -97,7 +97,7 @@
     it "does not allow slurped parameters" do
       expect do
         eval_template_with_args("<%-|*$x|-%><%= $x %>", 'x' => 'incorrect')
-      end.to raise_error(/'captures rest' - not supported in an Epp Template/)
+      end.to raise_error(/'captures rest', which is not supported in an expression of type Epp Template/)
```

Comment:
the `Epp Template` is an argument, and the sentence can look weird depending on the name of the expression type if it is inserted without the preamble `expression of type`.
It may also be confusing to a user if we are talking about the type of a data value, or the type of a programming language element - they are both upper case names - how is a user to know what the message is really saying?